### PR TITLE
[Hub Generated] Review request for Microsoft.DigitalTwins to add version preview/2020-03-01-preview

### PR DESCRIPTION
**Pull request by NateLehman**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/NateLehman/azure-rest-api-specs/dev-digitaltwins-Microsoft.DigitalTwins-remove-integrationresource...master/).

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by sup-s — APPROVED** [review]
We are removing apis that will not be supported for public preview

**Comment by NateLehman** [comment]
@njuCZ I believe this PR is ready for ARM review. Is it waiting to be picked up by the ARM reviewer?

File: specification/digitaltwins/resource-manager/Microsoft.DigitalTwins/preview/2020-03-01-preview/digitaltwins.json [hunk 1]
```diff
@@ -512,171 +512,6 @@
         }
       }
     },
-    "/{scope}/providers/Microsoft.DigitalTwins/integrationResources/{integrationResourceName}": {
```
Comment (ravbhatnagar) [inline specification/digitaltwins/resource-manager/Microsoft.DigitalTwins/preview/2020-03-01-preview/digitaltwins.json:515]: removing a bunch of APIs - breaking change for preview customers. Their existing scenarios will be broken.
Do we have data on the usage of these APIs? OK to break them?

**Review by ravbhatnagar — COMMENTED** [review]
seems like this is deliberate, wanted to confirm about current customers using this? @NateLehman ?

**Comment by NateLehman** [comment]
@ravbhatnagar Yes, it’s ok to break. Those APIs are being cut from our public preview which is going live at the end of this month. There is not usage data. @sup-s is RP owner for DigitalTwins let us know if you need any info

**Pull request merged**